### PR TITLE
Update translation_ru.json

### PR DESCRIPTION
**Pull request by onxblog**
Translated last ENG functions

Please try and fill out this template where possible, not all fields are required and can be removed.

* **Please check if the PR fulfills these requirements**
- [] The changes have been tested locally
- [] There are no breaking changes

* **What kind of change does this PR introduce?**
(Bug fix, feature, docs update, ...)

* **What is the current behavior?**
(You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: Translation Editor/translation_ru.json [hunk 1]
```diff
@@ -247,31 +247,31 @@
 		},
 		"PowerPulsePower": {
 			"text2": [
-				"Power",
-				"Pulse W"
+				"Питание",
+				"удержать пит."
```
Comment (paulfertser) [inline Translation Editor/translation_ru.json:251]: These two lines are supposed to be visible at the same time. I'm not sure what the second "пит." means but if it's the same word as "Питания" it'd be really odd IMHO.

File: Translation Editor/translation_ru.json [hunk 2]
```diff
@@ -247,31 +247,31 @@
...
 			],
-			"desc": "Keep awake pulse power intensity"
+			"desc": "Интенсивность импульса удерживающего от сна источник питания"
```
Comment (paulfertser) [inline Translation Editor/translation_ru.json:253]: Missing comma.

File: Translation Editor/translation_ru.json [hunk 3]
```diff
@@ -247,31 +247,31 @@
...
 			],
-			"desc": "Sensitivity of the Hall effect sensor in detecting sleep <O=Off,L=Low,M=Medium,H=High>"
+			"desc": "Чувствительность датчика холла в режиме сна <O=Выкл,L=Низкий,M=Средний,H=Высокий>"
 		},
 		"LockingMode": {
 			"text2": [
-				"Allow buttons",
-				"locking"
+				"Кнопки активны",
+				"заблокировать"
```
Comment (paulfertser) [inline Translation Editor/translation_ru.json:272]: Three English words here form a sentence. Three Russian words seem to be uncoordinated, it's not a grammatically correct sentence AFAICT.

File: Translation Editor/translation_ru.json [hunk 4]
```diff
@@ -247,31 +247,31 @@
...
 		},
 		"HallEffSensitivity": {
 			"text2": [
-				"Hall Eff",
-				"Sensitivity"
+				"Эффект Холла",
+				"чувствительность"
 			],
-			"desc": "Sensitivity of the Hall effect sensor in detecting sleep <O=Off,L=Low,M=Medium,H=High>"
+			"desc": "Чувствительность датчика холла в режиме сна <O=Выкл,L=Низкий,M=Средний,H=Высокий>"
```
Comment (paulfertser) [inline Translation Editor/translation_ru.json:267]: Grammatical gender seems to be mixed up here.

File: Translation Editor/translation_ru.json [hunk 5]
```diff
@@ -247,31 +247,31 @@
...
 		},
 		"LockingMode": {
 			"text2": [
-				"Allow buttons",
-				"locking"
+				"Кнопки активны",
+				"заблокировать"
 			],
-			"desc": "When soldering, long press on both buttons lock them <D=Disable, B=Boost only, F=Full locking>"
+			"desc": "При работе, длинное нажатие обеих кнопок блокирует их <D=Отключено, B=Только турбо, F=Полная блокировка>"
```
Comment (paulfertser) [inline Translation Editor/translation_ru.json:274]: Extra comma.

File: Translation Editor/translation_ru.json (hunk 1 above)
Comment (onxblog) [reply Translation Editor/translation_ru.json:251]: I'm not really sure about this function "Pulse W" 
What is it really mean ?

**Review by paulfertser — COMMENTED** [review]
Hello. I'm not the target audience of this translation but I have some familiarity with the language so hope my comments can be useful. That said, feel free to ignore them, I wouldn't mind.

File: Translation Editor/translation_ru.json (hunk 3 above)
Comment (onxblog) [reply Translation Editor/translation_ru.json:272]: OK. I saw my mistakes.

File: Translation Editor/translation_ru.json (hunk 1 above)
Comment (paulfertser) [reply Translation Editor/translation_ru.json]: This allows to set the power (in Watts) of the short periodic pulse (it's short and rare enough to not heat the iron considerably but should be big enough to keep the power supply awake).

**Comment by paulfertser** [comment]
BTW, please do not forget to test the translation on the device to ensure all new menu lines fit the screen.

File: Translation Editor/translation_ru.json (hunk 4 above)
Comment (onxblog) [reply Translation Editor/translation_ru.json:267]: No it's all ok. For me.

File: Translation Editor/translation_ru.json (hunk 5 above)
Comment (onxblog) [reply Translation Editor/translation_ru.json]: fixed

File: Translation Editor/translation_ru.json (hunk 4 above)
Comment (paulfertser) [reply Translation Editor/translation_ru.json:267]: "Чувствительность" is a noun with a female grammatical gender. "Низкий/Средний/Высокий" are in male grammatical gender. I would expect them to match.

Comment (onxblog) [reply Translation Editor/translation_ru.json:267]: Ok I fix it

**Comment by Ralim** [comment]
Heya; where is this up to?
Is this in a mergable state? (I have zero ability to read this I'm afraid) 😢

**Comment by onxblog** [comment]
> Heya; where is this up to?
> Is this in a mergable state? (I have zero ability to read this I'm afraid) 😢

Hi. Last update is correct with syntax and grammar and gender words.
Translation conveys the meaning 100%
If my translation is appropriate, you can merge it.

**Pull request merged**